crawl-ref/source/dat/descript/monsters.txt

```diff
@@ -801,11 +801,24 @@ corrupter
 
 <demonspawn>
 %%%%
+creeping inferno
+
+A formless mass of ancient fire, bound by heat and malevolence. The slightest
+disturbance will destroy the bonds holding it together and unleash a fire
+storm. It longs for that release yet it seems somehow reluctant to move while
+being observed.
+%%%%
 crimson imp
 
 A small, ugly minor demon. Its fanged maw seems locked in a permanent malign
 grin, and its horns protrude obscenely from its scaly pate.
 %%%%
+crystal echidna
+
+A spiny, long snouted rodent grown out of pure living crystal. It is
```

Comment:
...should we have some explanation for why a rodent is lurking in hell, or are we just going to roll with it?

Reply:
I added `serial_glass` to Dis except it places green crystal instead of glass. Might also add green crystal to some of the layouts. Beyond the terrain "explanation" I don't think we need to say more: it's a city, it has vermin!